tools/autograd/derivatives.yaml

```diff
@@ -295,6 +295,9 @@
 - name: any.dim(Tensor self, int dim, bool keepdim=False) -> Tensor
   output_differentiability: [False]
 
+- name: any.dims(Tensor self, int[1]? dim=None, bool keepdim=False) -> Tensor
+  output_differentiability: [False]
+
```

Comment:
Can't you make it composite implicit, even if it's not differentiable?

Reply:
No, because that would have higher priority than the `CPU` and `CUDA` dispatch.